meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -1,7 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 BRANCH_lmp = "master"
-SRCREV_lmp = "fe447e0c56d05d9d003722a2bbdc9a00205f6a55"
+SRCREV_lmp = "1cfc6e6dbc7aa358effc7334c6f861dc30c2332e"
```

Comment:
No need to update aktualizr, latest is already in meta-lmp now.

Reply:
fixed.